mod_reforged/hooks/skills/effects/adrenaline_effect.nut

```diff
@@ -0,0 +1,37 @@
+::Reforged.HooksMod.hook("scripts/skills/effects/adrenaline_effect", function(q) {
+	q.m.GrantsInjuryImmunity <- true;
+
+	q.create = @(__original) function()
+	{
+		__original();
+
+		this.m.Order = ::Const.SkillOrder.PermanentInjury - 100;	// This effect needs to flip the IsAffectedByInjuries flag before any of the injuries had the time to inflict their effects
```

Comment:
Why the `-100`? Setting it to PermanentInjury seems good enough.

Reply:
I felt putting it slightly before permanent injuries was the safest option just in case we want it to trigger before permanent injuries at some point.

Reply:
Permanent Injuries are not affected by `IsAffectedByInjuries` flag. But it's fine to set the order as you have done for future proofing.

Reply:
Thinking more about it I changed it from permanentInjury to temporaryInjury.

Adrenaline should NEVER affect permanent injuries, so I can't think of any scenario where that order should ever be relevant

Reply:
Yes that's fine by me.